pkg/resolution/resolver/bundle/resolver_test.go

```diff
@@ -169,6 +191,347 @@ func TestResolveDisabled(t *testing.T) {
+
+			confMap := map[string]string{
+				ConfigKind:           "task",
+				ConfigServiceAccount: "myksa",
+			}
+
+			d := test.Data{
+				ResolutionRequests: []*v1beta1.ResolutionRequest{request},
+				ConfigMaps: []*corev1.ConfigMap{{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      ConfigMapName,
+						Namespace: resolverconfig.ResolversNamespace(system.Namespace()),
+					},
+					Data: confMap,
+				}, {
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: resolverconfig.ResolversNamespace(system.Namespace()),
+						Name:      resolverconfig.GetFeatureFlagsConfigName(),
+					},
+					Data: map[string]string{
+						"enable-bundles-resolver": "true",
+					},
+				}},
+			}
+			var expectedStatus *v1beta1.ResolutionRequestStatus
+			var expectedError error
+			if tc.expectedStatus != nil {
+				expectedStatus = tc.expectedStatus.DeepCopy()
+				if tc.expectedErrMessage == "" {
+					reqParams := make(map[string]string)
+					for _, p := range request.Spec.Params {
+						reqParams[p.Name] = p.Value.StringVal
+					}
+					if expectedStatus.Annotations == nil {
+						expectedStatus.Annotations = make(map[string]string)
+					}
+
+					if reqParams[ParamKind] != "" {
+						expectedStatus.Annotations[ResolverAnnotationKind] = reqParams[ParamKind]
+					} else {
+						expectedStatus.Annotations[ResolverAnnotationKind] = confMap[ConfigKind]
+					}
+
+					expectedStatus.Annotations[ResolverAnnotationName] = reqParams[ParamName]
+					expectedStatus.Annotations[ResolverAnnotationAPIVersion] = "v1beta1"
+				} else {
+					expectedError = createError(ref, tc.expectedErrMessage)
+					expectedStatus.Status.Conditions[0].Message = expectedError.Error()
+				}
+			}
+
+			frtesting.RunResolverReconcileTest(ctx, t, d, resolver, request, expectedStatus, expectedError)
+
+		})
+	}
+}
+
+func createRequest(serviceAccount, bundle, name, kind string) *v1beta1.ResolutionRequest {
+	rr := &v1beta1.ResolutionRequest{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "resolution.tekton.dev/v1beta1",
+			Kind:       "ResolutionRequest",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              "rr",
+			Namespace:         "foo",
+			CreationTimestamp: metav1.Time{Time: time.Now()},
+			Labels: map[string]string{
+				resolutioncommon.LabelKeyResolverType: LabelValueBundleResolverType,
+			},
+		},
+		Spec: v1beta1.ResolutionRequestSpec{
+			Params: []pipelinev1beta1.Param{{
+				Name:  ParamBundle,
+				Value: *pipelinev1beta1.NewStructuredValues(bundle),
+			}, {
+				Name:  ParamName,
+				Value: *pipelinev1beta1.NewStructuredValues(name),
+			}, {
+				Name:  ParamKind,
+				Value: *pipelinev1beta1.NewStructuredValues(kind),
+			}, {
+				Name:  ParamServiceAccount,
+				Value: *pipelinev1beta1.NewStructuredValues(serviceAccount),
+			}},
+		},
+	}
+	return rr
+}
+
+func createResolutionRequestStatus(content []byte) *v1beta1.ResolutionRequestStatus {
```

Comment:
there is a `createStatus` in resolver/git and I think the `createResolutionRequestStatus` is better. Maybe we can think of rename createStatus or consolidate them into one?

Reply:
Good catch! consolidated them into one named `CreateResolutionRequestStatusWithData` and moved into the `pkg/resolution/resolver/framework/testing/fakecontroller.go` so that it can be shared in unit tests across different resolvers.

PTAL. Thanks!